src/Data/Rational/Properties.agda

```diff
@@ -1719,6 +1720,91 @@ toℚᵘ-homo-∣-∣ (mkℚ -[1+ _ ] _ _) = *≡* refl
 ∣∣p∣∣≡∣p∣ : ∀ p → ∣ ∣ p ∣ ∣ ≡ ∣ p ∣
 ∣∣p∣∣≡∣p∣ p = 0≤p⇒∣p∣≡p (0≤∣p∣ p)
 
+------------------------------------------------------------------------
+-- Metric space
+------------------------------------------------------------------------
+
+private
+  d : ℚ → ℚ → ℚ
+  d p q = ∣ p - q ∣
+
+d-cong : Congruent _≡_ d
+d-cong = cong₂ _
+
+d-nonNegative : ∀ {p q} → 0ℚ ≤ d p q
+d-nonNegative {p} {q} = nonNegative⁻¹ _ {{∣-∣-nonNeg (p - q)}}
+
+d-definite : Definite _≡_ d
+d-definite {p} refl = cong ∣_∣ (+-inverseʳ p)
+
+d-indiscernable : Indiscernable _≡_ d
+d-indiscernable {p} {q} ∣p-q∣≡0 = begin
+  p               ≡˘⟨ +-identityʳ p ⟩
```

Comment:
Obviously it would be great if we could use the new symmetric reasoning syntax.

Reply:
I continue to strongly dislike the new symmetric reasoning syntax. emacs treats `⟨` and `⟩` as brackets, which means if I miss a `)` (which I often do if I write a whole sequence of reasoning in one go) I can't use emacs' bracket-matching to find where it should go

Reply:
We have officially deprecated the old syntax: https://github.com/agda/agda-stdlib/blob/45a46f7691c328364856eeda8b042bf6f389d8a8/src/Relation/Binary/Reasoning/Syntax.agda#L270

It's just that Agda doesn't transfer deprecation warnings properly for parametrised modules.